.github/workflows/tests-gpu.yml

```diff
@@ -83,6 +84,18 @@ jobs:
           COVERALLS_PARALLEL: true
         run: coveralls --service=github
 
+  pyflakes:
+    name: runner/pyflakes
+    needs: cases
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: pyflakes
+        uses: reviewdog/action-pyflakes@master
```

Comment:
I would prefer not to use a complex third party tool like reviewdog and to just run pyflakes directly ourselves.

Reviewdog (and similar tools like CodeClimate) add complexity for little benefit and some significant downsides (e.g. it becomes much harder to answer a simple question like "which version of pyflakes with what configuration is expected to work".

Reply:
Good point what do you think of using an scheme like the one on mitiq https://github.com/unitaryfund/mitiq/blob/master/.github/workflows/test.yml ? Maybe the calls to make overcomplexify it a ittle bit an wer can directly make the calls on the yml

Reply:
Let's try put the calls to pyflakes and black directly into the .yml to start with. If things get complicated, we can think about adding something like `make`.